src/main.css

```diff
@@ -4700,82 +4700,60 @@ a[href="#s"]:hover::after, a[href="#s"]:active::after {
 
 /* Flag tags */
 .md a[href*="#flags-"] {
-	background: url(%%tags-flags%%);
-	cursor: default;
-	display: inline-block;
-	font-size: 0;
-	height: 11px;
-	pointer-events: none;
-	width: 16px;
+	background:url(%%flags-all%%);
+	cursor:default;
+	display:inline-block;
+	font-size:;
+	height:15px;
+	pointer-events:none;
+	width:21px;
+}
+.md a[href="#flags-au"] {background-position: -21px -15px}
+.md a[href="#flags-be"] {background-position: -168px -15px}
+.md a[href="#flags-br"] {background-position: -168px -30px}
+.md a[href="#flags-bs"] {background-position: -189px -30px}
+.md a[href="#flags-by"] {background-position: -63px -45px}
+.md a[href="#flags-ca"] {background-position: -105px -45px}
+.md a[href="#flags-cn"] {background-position: -105px -60px}
+.md a[href="#flags-de"] {background-position: -84px -75px}
+.md a[href="#flags-dk"] {background-position: -126px -75px}
+.md a[href="#flags-es"] {background-position: -84px -390px}
+.md a[href="#flags-eu"] {background-position: -63px -90px}
+.md a[href="#flags-fi"] {background-position: -84px -90px}
+.md a[href="#flags-fr"] {background-position: -189px -90px}
+.md a[href="#flags-gb"] {background-position: -21px -105px}
+.md a[href="#flags-hu"] {background-position: -189px -135px}
+.md a[href="#flags-is"] {background-position: -168px -150px}
+.md a[href="#flags-it"] {background-position: -189px -150px}
+.md a[href="#flags-jp"] {background-position: -63px -165px}
+.md a[href="#flags-kr"] {background-position: -21px -180px}
+.md a[href="#flags-lt"] {background-position: -63px -195px}
+.md a[href="#flags-lv"] {background-position: -105px -195px}
+.md a[href="#flags-nl"] {background-position: -105px -240px}
+.md a[href="#flags-no"] {background-position: -126px -240px}
+.md a[href="#flags-pt"] {background-position: -63px -270px}
+.md a[href="#flags-ro"] {background-position: -168px -270px}
+.md a[href="#flags-ru"] {background-position: 0px -285px}
+.md a[href="#flags-se"] {background-position: -147px -285px}
+.md a[href="#flags-th"] {background-position: -21px -330px}
+.md a[href="#flags-ua"] {background-position: -84px -345px}
+.md a[href="#flags-us"] {background-position: -126px -345px}
+.md a[href="#flags-ww"] {background-position: -189px -360px}
+.md a[href="#flags-?-ow"] {background-position: -42px -390px}
+
+/* Result Background */
+.md a[href^="#resultbg"] {
+  	box-sizing: border-box;
```

Comment:
Yeah, it does feel a bit like overkill, but I just found it difficult to make something written in text stand out when there are also tables in the post, since they are pretty "high profile" here. /r/soccer gace me the idea for this actually 😃